pilot/helpers/test_Project.py

```diff
@@ -50,14 +51,10 @@ def test_get_full_path(self, file_path, file_name, expected):
         {'name': None, 'path': 'public/index.html', 'saved_to': f'{test_root}/public/index.html'},
         {'name': '', 'path': 'public/index.html', 'saved_to': f'{test_root}/public/index.html'},
         # TODO: Treatment of paths outside of the project workspace - https://github.com/Pythagora-io/gpt-pilot/issues/129
-        # {'name': '/etc/hosts', 'path': None, 'saved_to': '/etc/hosts'},
+        {'name': '/etc/hosts', 'path': None, 'saved_to': f'{test_root}/etc/hosts'},

```

Comment:
Basically this means that we never reach out outside the project directory (for now). Any absolute paths are treated as relative. I think this is the safest option until we come to the point where we really want to mess with the rest of the user's system (which we should do really carefully, like ask for every overwrite or maybe even read)